sdk/ml/azure-ai-ml/azure/ai/ml/operations/_component_operations.py

```diff
@@ -167,6 +167,36 @@ def list(
             )
         )
 
+    @monitor_with_telemetry_mixin(logger, "ComponentVersion.Get", ActivityType.PUBLICAPI)
+    def get_component_version(self, name: str, version: Optional[str] = DEFAULT_COMPONENT_VERSION) -> ComponentVersion:
```

Comment:
the same here. public API change should be reviewed carefully.

For this one, ComponentVersion is not a concept we would want to expose to customer. Also, user may feel confused given there is already a `get`

Reply:
yes.